src/main/java/com/crosswordservice/crosswordchecker/GenerateCrossword.java

```diff
@@ -0,0 +1,180 @@
+package com.crosswordservice.crosswordchecker;
+
+import com.crosswordservice.baseClasses.Crossword;
+import com.crosswordservice.baseClasses.Intersection;
+import com.crosswordservice.baseClasses.Question;
+
+import java.util.*;
+
+public class GenerateCrossword {
```

Comment:
I am also missing a bit more documentation here (even if they are most private functions). It is such a big file.